src/foam/core/Unit.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.core',
+  name: 'Unit',
+  //abstract: true,
+
+  documentation: `The abstract model for fungible digitized assets`,
+
+  ids: [
+    'alphabeticCode'
```

Comment:
Why wouldn't we just rename `alphabeticCode` to `id`? I don't think `alphabeticCode` makes much sense as a name, whereas `id` is consistent with the rest of the system.

Reply:
yes, we need to do this.

Reply:
changed this to id.